scripts/eval/yamls/eval_gauntlet.yaml

```diff
@@ -115,14 +115,58 @@ eval_gauntlet:
   - name: programming
     benchmarks:
     - name: human_eval
-      num_fewshot: 0
+      num_fewshot: [0, 3]
```

Comment:
should just be "3"

Reply:
Resolution here is to go with 0 as is consistent with HumanEval methodology in other papers (e.g., CodeLlama, StarCoder, etc)

Reply:
cosigned